airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeGcsStreamCopier.java

```diff
@@ -19,40 +19,55 @@
 import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.StringJoiner;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class SnowflakeGcsStreamCopier extends GcsStreamCopier {
+public class SnowflakeGcsStreamCopier extends GcsStreamCopier implements SnowflakeParallelCopyStreamCopier {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SnowflakeGcsStreamCopier.class);
+  private final ExecutorService executorService = Executors.newFixedThreadPool(5);
```

Comment:
nitpick: I think this executor service doesn't need to be instantiated immediately, i.e. it can just be a variable inside `copyFilesInParallel`?

Reply:
> nitpick: I think this executor service doesn't need to be instantiated immediately, i.e. it can just be a variable inside `copyFilesInParallel`?

updated `copyFilesInParallel` method with local variable executorService